### ISSUE TITLE
Rework fallback message

### ISSUE THREAD

**Issue by olegs** [JetBrains-Research/viktor#10]
Current version:

```
Native SIMD optimization of array operations is not available.
Fallback Kotlin implementation will be used instead.
Build viktor for your system from source as described in https://github.com/JetBrains-Research/viktor
```

I think that there are 0 people in the world, who will use zinbra and will be able to rebuild viktor from sources. The audience is literally empty. So I'd rather prefer something like this:
```SIMD computation optimisations are not supported for your system, use --debug for details.```
And provide some extended message why this is happening with DEBUG log level, like:
```
SSE, AVX not available on your system. 
Fallback Kotlin implementation will be used instead.
Build viktor for your system from source as described in https://github.com/JetBrains-Research/viktor
```

**Comment by dievsky** [comment]
Released updated Viktor on BinTray as 0.4.4, updated Epigenome accordingly.

**Comment by dievsky** [comment]
Released updated viktor with correct logging on BinTray as 0.4.5, updated Epigenome. Judging by TeamCity output on Windows agent, the message is displayed as intended.
